### PR TITLE
Developer rep

### PR DESCRIPTION
**Pull request by JordoBrooks**
Developers have reputation that can be increased by successfully completing project milestones.

### CODE REVIEW

File: smart-contracts/ethereum/contracts/FundingService.sol [hunk 1]
```diff
@@ -1,13 +1,11 @@
 pragma solidity ^0.4.23;
 import "./Project.sol";
-import "./SafeMath.sol";
```
Comment (ericmuyser) [inline smart-contracts/ethereum/contracts/FundingService.sol]: Note: SafeMath was removed temporarily as it's causing compilation issues (out of gas)

File: smart-contracts/ethereum/contracts/Project.sol [hunk 2]
```diff
@@ -43,6 +43,8 @@ contract Project {
 
     enum Status {Draft, Pending, Published, Removed, Rejected}
```
Comment (ericmuyser) [inline smart-contracts/ethereum/contracts/Project.sol]: I think we wanted to make these `uint` right? Maybe create an issue as not to forget

File: smart-contracts/ethereum/test/TestFundingService.js [hunk 3]
```diff
@@ -45,9 +45,10 @@ contract('FundingService', function(accounts) {
            assert.equal(newDeveloperId, 1, "New developer should be in developerMap with id = 1");
 
            const createdDeveloper = await fundingService.developers.call(newDeveloperId);
+
            assert.equal(createdDeveloper[0].toNumber(), newDeveloperId, "Incorrect ID");
-           assert.equal(createdDeveloper[1], devAccount, "Incorrect address");
-           assert.equal(createdDeveloper[2], devName, "Incorrect name");
+           assert.equal(createdDeveloper[2], devAccount, "Incorrect address");
+           assert.equal(createdDeveloper[3], devName, "Incorrect name");
```
Comment (ericmuyser) [inline smart-contracts/ethereum/test/TestFundingService.js]: The above uses `.toNumber()` but these don't?

**Comment by ericmuyser** [comment]
Looks good aside from minor feedback

File: smart-contracts/ethereum/contracts/Project.sol (hunk 2 above)
Comment (JordoBrooks) [reply smart-contracts/ethereum/contracts/Project.sol]: @ericmuyser Were we wanting to change these, too? I thought it was the Terms enum that we wanted to refactor (I ended up using boolean flags for them)

File: smart-contracts/ethereum/test/TestFundingService.js (hunk 3 above)
Comment (JordoBrooks) [reply smart-contracts/ethereum/test/TestFundingService.js]: createdDeveloper[0] is the developer ID which comes back as a BigNumber. The other two are just strings.

Comment (JordoBrooks) [reply smart-contracts/ethereum/test/TestFundingService.js]: createdDeveloper[0] returns the developer ID as a BigNumber. The other two are just an address (which returns fine for some reason) and a string.

File: smart-contracts/ethereum/contracts/Project.sol (hunk 2 above)
Comment (ericmuyser) [reply smart-contracts/ethereum/contracts/Project.sol]: Ah I was thinking we may abandon enum entirely for the casting issues. Gut feeling is we'd switch at some point, but lets benefit of the doubt for now

**Pull request merged**